<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReplaceWindow</class>
 <widget class="QDialog" name="ReplaceWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>68</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>替换</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="string_edit">
     <property name="placeholderText">
      <string>被替换...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="replace_now_btn">
     <property name="text">
      <string>替换当前</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="now_edit">
     <property name="placeholderText">
      <string>替换成...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="replace_all_btn">
     <property name="text">
      <string>替换全部</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
